examples/step-63/doc/results.dox

```diff
@@ -1,2 +1,440 @@
+  <td>7</th>
+  <td></th>
+  <th>4800</th>
+  <td>29</th>
+  <td>21</th>
+  <td>28</th>
+  <td>9</th>
+</tr>
+<tr>
+  <th>2048</th>
+  <td></th>
+  <th>2176</th>
+  <td>15</th>
+  <td>13</th>
+  <td>10</th>
+  <td>8</th>
+  <td></th>
+  <th>18816</th>
+  <td>33</th>
+  <td>22</th>
+  <td>32</th>
+  <td>9</th>
+</tr>
+<tr>
+  <th>8192</th>
+  <td></th>
+  <th>8448</th>
+  <td>18</th>
+  <td>15</th>
+  <td>11</th>
+  <td>10</th>
+  <td></th>
+  <th>74496</th>
+  <td>35</th>
+  <td>22</th>
+  <td>34</th>
+  <td>10</th>
+</tr>
+<tr>
+  <th>32768</th>
+  <td></th>
+  <th>33280</th>
+  <td>20</th>
+  <td>16</th>
+  <td>12</th>
+  <td>10</th>
+  <td></th>
+</tr>
+<tr>
+  <th>131072</th>
+  <td></th>
+  <th>132096</th>
+  <td>20</th>
+  <td>16</th>
+  <td>12</th>
+  <td>11</th>
+  <td></th>
+</tr>
+</table>
+
+We see that for $Q_1$, both multiplicative smoothers require a smaller combination of smoothing steps and iteration counts than either additive smoother. However, when we in increase the degree to a $Q_3$ element, there is a clear advantage for the block smoothers in terms of the increase in smoothing steps and iterations required to solve. Specifically, the block SOR smoother gives constant iterations over degree, and the block Jacobi only sees about a 38% increase in iterations compared to 75% and 183% for Jacobi and SOR respectively.
+
+<h3> Cost </h3>
+
+Iteration counts do not tell the full story in the optimality of a one smoother over another. Obviously we must examine the cost of an iteration. Block smoothers here are at a disadvantage as they are having to construct and invert a cell matrix for each cell. Here is a comparison of solve times for a $Q_3$ element with 74,496 DoFs:
+ 
+<table align="center" border="1">
+<tr>
+  <th colspan="1">$Q_3$</th>
+  <th colspan="4">Smoother (smoothing steps)</th>
+</tr>
+<tr>
+  <th colspan="1">DoFs</th>
+  <th colspan="1">Jacobi (6)</th>
+  <th colspan="1">Block Jacobi (3)</th>
+  <th colspan="1">SOR (3)</th>
+  <th colspan="1">Block SOR (1)</th>
+</tr>
+<tr>
+  <th>74496</th>
+  <td>0.689 s</th>
+  <td>5.826 s</th>
+  <td>1.189 s</th>
+  <td>1.021 s</th>
+</tr>
+</table>
+
+The smoother that requires the most iterations (Jacobi) actually take the shortest time (roughly 2/3 the time of the next fastest method). This is because all that is requires to apply a Jacobi smoothing step is multiplication by a diagonal matrix which is very cheap. On the other hand, while SOR requires over 3x more iterations (each with 3x more smoothing steps) than block SOR, the times are roughly equivalent, implying that a smoothing step of block SOR is roughly 9x slower than a smoothing step of SOR. Lastly, block Jacobi is almost 6x more expensive than block SOR, which intuitively makes sense from the fact that 1 step of each method has the same cost (inverting the cell matrices and either adding or multiply them together), and block Jacobi has 3 times the smoothing step per iteration with 2 times the iterations.
```

Comment:
```suggestion
The smoother that requires the most iterations (Jacobi) actually takes the shortest time (roughly 2/3 the time of the next fastest method). This is because all that is required to apply a Jacobi smoothing step is a multiplication by a diagonal matrix which is very cheap. On the other hand, while SOR requires over 3x more iterations (each with 3x more smoothing steps) than block SOR, the times are roughly equivalent, implying that a smoothing step of block SOR is roughly 9x slower than a smoothing step of SOR. Lastly, block Jacobi is almost 6x more expensive than block SOR, which intuitively makes sense from the fact that 1 step of each method has the same cost (inverting the cell matrices and either adding or multiply them together), and block Jacobi has 3 times the smoothing step per iteration with 2 times the iterations.
```